static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.ts

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable @typescript-eslint/no-misused-promises
+-- Fastdom measure and mutate signatures are Promise<void>
+-- Nested fastdom measure-mutate promises throw the error:
+-- "Promise returned in function argument where a void return was expected"
+*/
```

Comment:
This file has been converted to TypeScript.

Main for reference:

https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.js